engine/src/main/java/org/hibernate/validator/internal/metadata/raw/ConstrainedExecutable.java

```diff
@@ -257,8 +257,8 @@ public boolean isEquallyParameterConstrained(ConstrainedExecutable other) {
 		int i = 0;
 		for ( ConstrainedParameter parameter : parameterMetaData ) {
 			ConstrainedParameter otherParameter = other.getParameterMetaData( i );
-			if ( parameter.isCascading != otherParameter.isCascading || !getDescriptors( parameter.getConstraints() )
-					.equals( getDescriptors( otherParameter.getConstraints() ) ) ) {
+			if ( parameter.isCascading != otherParameter.isCascading 
```

Comment:
It's a pity to have this pure formatting changes in this PR. But I won't go back to sort that out :/